templates/zerver/documentation_main.html

```diff
@@ -10,7 +10,8 @@
 <div class="app help terms-page inline-block{% if page_is_help_center %} help-center{% endif %}{% if page_is_api_center %} api-center{% endif %}">
     <div class="sidebar">
         <div class="content">
-            <h1><a href="{{ doc_root }}" class="no-underline">Home</a></h1>
+            <h1 class="home-link-top"><a href="https://zulip.com/" class="no-underline">Go to Zulip.com</a></h1>
+            <h1><a href="{{ doc_root }}" class="no-underline">Home - {{ doc_root_title }}</a></h1>
```

Comment:
I don't think this looks super good.  What if we just made the `Zulip |` part of the top-left index go to zulip.com?

![image](https://user-images.githubusercontent.com/2746074/124205012-317a1a00-da95-11eb-8d3e-29c5cd3d784c.png)

There's also a question of whether zulip.com is the right link on a self-hosted server, or whether we want it to be `href=/` (i.e. go to the root of the current server you're looking at).

Reply:
What would `href=/` do, exactly? The goal is for it to be easy to get back to zulip.com if you navigate to help through the top menus.

Reply:
@timabbott: I linked to `/` earlier but changed it after Alya suggested that we go straight to zulip.com.

@alya: Linking to `href=/` takes you to the root of the current server for self-hosted servers. For our zulip.com domain, linking to `href=/` would still take you to the main website, so it actually doesn't matter for our purposes. But I think I agree with Tim here, it makes sense to make the link work more predictably for self-hosted servers, so I'll change it to `href=/`. 

Thanks!

Reply:
OK, after some discussion, I think we should just modify master in this spot by replacing "Home" with a "Zulip homepage" entry that links to zulip.com.  If we leave the "Zulip Help Center" link at the top as-is, that's a fine way to get back to the index for those who want that; not sure we need it duplicated in the sidebar.

(Also, I think this shouldn't have been marked as resolved, since there's substantial discussion that hadn't been read by others)